packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -514,6 +515,11 @@ async function setupOrgBrowser(
       await forceSourceRetrieveCmp(trigger, true);
     }
   );
+
+  vscode.commands.registerCommand(
+    'sfdx.create.manifest',
```

Comment:
Should this be an enum or constant? Having all the commands we support for a particular module live in one place seems like a win.

Reply:
Created here: W-10522606

Reply:
👍